lib/rubocop/rspec/factory_bot.rb

```diff
@@ -4,7 +4,7 @@ module RuboCop
   module RSpec
     # RuboCop FactoryBot project namespace
     module FactoryBot
-      ATTRIBUTE_DEFINING_METHODS = %i[factory trait transient ignore].freeze
+      ATTRIBUTE_DEFINING_METHODS = %i[factory trait traits_for_enum transient ignore].freeze
```

Comment:
This list doesn't seem to be ordered alphabetically, so I decided to add the method near a similar entry, `:trait`.